Plugins/Json/src/SurfaceJsonConverter.cpp

```diff
@@ -25,6 +27,15 @@
 #include "Acts/Surfaces/StrawSurface.hpp"
 #include "Acts/Surfaces/TrapezoidBounds.hpp"
 
+void Acts::to_json(nlohmann::json& j, const Acts::SurfaceAndMaterial& surface) {
+  to_json(j, surface.first);
```

Comment:
shouldn't this be `(*surface.first)` ?
I only impelemted the surface converter for reference and `shared_ptr`